control-plane/pkg/reconciler/source/source.go

```diff
@@ -116,7 +116,7 @@ func (r Reconciler) reconcileConsumerGroup(ctx context.Context, ks *sources.Kafk
 				BackoffDelay:  pointer.String("PT0.3S"),
 				Timeout:       pointer.String("PT600S"),
 			},
-			Ordering: DefaultDeliveryOrder,
+			Ordering: *ks.Spec.Ordering,
```

Comment:
Same here, I guess we can do it once for both branches

Reply:
added the logic check back.